app/views/followings/index.html.haml

```diff
@@ -0,0 +1,23 @@
+%section.people
+  %header
+    %h2 People I Follow
+  %table.table
+    %thead
+      %tr
+        %th(width="30%")
+        %th(width="20%") Videos in Queue
+        %th(width="20%") Followers
+        %th(width="30%") Unfollow
+    %tbody
+      - @followings.each do |following|
+        %tr
+          %td
+            %img(src="http://www.gravatar.com/avatar/#{Digest::MD5.hexdigest(following.followee.email.downcase)}?s=40")
```

Comment:
I would use `= image_tag gravatar_for ... ` method here, where I would make `gravatar_for` helper method in `ApplicationHelper`.